File: backend/src/main/java/ch/rafaelurben/sheetmusiclearner/backend/io/async/dto/request/PiecePermissionAddRequestDto.java
/* (C) 2026 - Rafael Urben */
package ch.rafaelurben.sheetmusiclearner.backend.io.async.dto.request;

import ch.rafaelurben.sheetmusiclearner.backend.api.dto.PermissionType;
import java.util.UUID;

public record PiecePermissionAddRequestDto(UUID userId, PermissionType permissionType) {}

File: frontend/src/interfaces/async/request/piece/PiecePermissionAddRequestDto.ts
/*
 * (C) 2026. - Rafael Urben
 */

import type { PermissionType } from "@/api/generated/openapi/models/PermissionType.ts";

export default interface PiecePermissionAddRequestDto {
  userId: string;
  permissionType: PermissionType;
}
